Assets/Scripts/ROS Scripts/ROS Connections v2/Sensors/PointCloudSensor_ROSSensorConnection.cs

```diff
@@ -35,7 +35,9 @@ public void InitilizeSensor(int uniqueID, string sensorIP, int sensorPort, List<
             pcVisualizer.SetParentTransform(this.transform);
             pcVisualizers.Add(subscriberTopic, pcVisualizer);
             Debug.Log("Point Cloud subscribing to: " + subscriberTopic);
-            ros.AddSubscriber("/voxblox_node/" + subscriber, this);
+
+            sensorSubscriberTopicsDict.Add(subscriberTopic, true);
+            ros.AddSubscriber(subscriberTopic, this);
```

Comment:
the subscriber topic won't work by itself, needs to have "/voxblox_node/" in front (simlarly with camera streaming sensors I'm working on)